base/linalg/lapack.jl

```diff
@@ -775,7 +993,41 @@ for (gesvx, elty, relty) in
             X, rcond, ferr, berr, rpgf
         end
     end
- end
+end
+
+"""
+    gesvx!(fact, trans, A, AF, ipiv, equed, R, C, B) -> (X, equed, R, C, B, rcond, ferr, berr, work)
+
+Solves the linear equation `A * X = B` (`trans = N`), `A.' * X =B`
+(`trans = T`), or `A' * X = B` (`trans = C`) using the `LU` factorization
+of `A`. `fact` may be `E`, in which case `A` will be equilibrated and copied
+to `AF`; `F`, in which case `AF` and `ipiv` from a previous `LU` factorization
+are inputs; or `N`, in which case `A` will be copied to `AF` and then
+factored. If `fact = F`, `equed` may be `N`, meaning `A` has not been
+equilibrated; `R`, meaning `A` was multiplied by `diagm(R)` from the left;
+`C`, meaning `A` was multiplied by `diagm(C)` from the right; or `B`, meaning
+`A` was multiplied by `diagm(R)` from the left and `diagm(C)` from the right.
+If `fact = F` and `equed = R,B` the elements of `R` must all be positive.
```

Comment:
Right, those are the allowable values for that input, but for this conditional sentence "the elements of `R` must all be positive," which values of `equed` does that apply to? I think an "or" would be clearer than a comma.